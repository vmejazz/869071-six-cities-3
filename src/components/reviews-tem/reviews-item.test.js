import React from "react";
import renderer from "react-test-renderer";
import ReviewsItem from "./reviews-item.jsx";

const review = {
  srcAvatar: ``,
  name: `Bob`,
  rate: 2,
  text: `Test mock text`,
  date: `2018-02-25`
};

it(`Review section render`, () => {
  const tree = renderer
    .create(
        <ReviewsItem
          review={review}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
