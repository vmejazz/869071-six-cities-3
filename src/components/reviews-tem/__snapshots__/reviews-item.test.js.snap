// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review section render 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height={54}
        width={54}
      />
    </div>
    <span
      className="reviews__user-name"
    />
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "NaN%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    />
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      February 2018
    </time>
  </div>
</li>
`;
